meilisearch/src/analytics/segment_analytics.rs

```diff
@@ -1811,6 +1825,9 @@ impl SimilarAggregator {
                    "avg_criteria_number": format!("{:.2}", filter_sum_of_criteria_terms as f64 / filter_total_number_of_criteria as f64),
                    "most_used_syntax": used_syntax.iter().max_by_key(|(_, v)| *v).map(|(k, _)| json!(k)).unwrap_or_else(|| json!(null)),
                 },
+                "vector": {
+                    "retrieve_vectors": retrieve_vectors,
```

Comment:
I would expect this to be under `hybrid` as well.

`hybrid` in the context of `similar` sounds illogical, but was chosen to avoid multiplying the number of properties

Reply:
Huum, but we already have a `vector` for the search. Shouldn’t we keep the sequence `vector.retrieve_vectors` everywhere for consistency when retrieving these properties through amplitude?
<img width="409" alt="image" src="https://github.com/meilisearch/meilisearch/assets/7032172/6db70453-2b9a-46a0-b93d-cda93c8d01af">
Or should I move both to `hybrid.retrieve_vectors`?
But then, it seems less logical since it has nothing to do with the hybrid search.

Maybe @macraig, if you have 2s to check this out, you have a preference as the consumer of these analytics?

Reply:
ah I didn't see it was under vector for the search, I had seen it under hybrid. Please disregard my comment then